src/commands/create-scene.ts

```diff
@@ -255,16 +293,78 @@ f79d4760-0b71-44e4-ad0b-22743fdd4ca3
         }
       }
 
-      if (sceneData) {
-        this.log(sceneData.id);
-
-        const getSceneItemsRes = await client.sceneItems.getSceneItems({
-          id: sceneData.id,
-          pageSize: 1,
-        });
-        if (getSceneItemsRes.data.data.length === 0) {
-          this.error(`No scene items exist in the scene.`);
+      if (sceneData && sceneData.id) {
+        const sceneId = sceneData.id;
+        if (experimental) {
+          if (verbose) cli.info(`Validating scene items...`);
+          let sceneItemCount = 0;
+          const reqItemMap: Record<string, SceneItem> = items.reduce<
+            Record<string, SceneItem>
+          >((itemMap, item) => {
+            itemMap[item.suppliedId] = item;
+            return itemMap;
+          }, {});
+          let cursor: string | undefined;
+          let itemsRemain = items.length > 0;
+          while (itemsRemain) {
+            // eslint-disable-next-line no-await-in-loop
+            const res = await getPage(() =>
```

Comment:
What happens here if there is an item that gets created on the first page, but we've already passed the first page in the pagination process?

Reply:
Discussed on Slack - recommending here to split this while loop into its own function that fetches all the items for readability.

Reply:
Addressed.